app/src/main/res/navigation/mobile_navigation.xml

```diff
@@ -35,4 +35,46 @@
         android:name="ch.epfl.sweng.rps.ui.game.GameFragment"
         android:label="@string/menu_game"
         tools:layout="@layout/fragment_game" />
+    
+    <fragment
+        android:id="@+id/nav_friends"
+        android:name="ch.epfl.sweng.rps.ui.friends.FriendsFragment"
+        android:label="Friends"
+        tools:layout="@layout/fragment_friends" >
+        <action
+            android:id="@+id/action_nav_friends_to_infoPage_Fragment"
+            app:destination="@id/infoPage_Fragment" />
+        <action
+            android:id="@+id/action_nav_friends_to_gameFragment"
+            app:destination="@id/gameFragment" />
+        <argument
+            android:name="userName"
+            app:argType="string" />
+        <argument
+            android:name="winRateText"
+            app:argType="string" />
+        <argument
+            android:name="gamesPlayedText"
+            app:argType="string" />
+        <argument
+            android:name="gamesWonText"
+            app:argType="string" />
+        <argument
+            android:name="isOnline"
+            app:argType="boolean" />
+    </fragment>
```

Comment:
Why do you pass all these strings arguments to the friend's page? Can't you get the data directly in the friends fragment from on of the data classes?